config.go

```diff
@@ -192,19 +188,95 @@ func loadConfiguration(configPath string) (oci.RuntimeConfig, ShimConfig, error)
 
 			config.ProxyType = vc.CCProxyType
 			config.ProxyConfig = pConfig
+
+			break
+		}
+	}
+
+	for k, agent := range tomlConf.Agent {
+		switch k {
+		case hyperstartAgent:
+			path := filepath.Join(agent.pauseBundlePath(),
+				pauseBinRelativePath)
```

Comment:
I agree it's best not to hard-code, but an OCI bundle contains a `config.json` and a rootfs directory. Conventionally the latter is called `rootfs/`, but it doesn't have to be. Strictly you need to check the `root.path` element in `config.json` to know the actual relative path to the bundles rootfs directory. But the code now seems to just glue the bundle path and the pause bin path together which looks wrong (previously you did have a `rootfs/` directory element).